core-backend/src/wasmi/env.rs

```diff
@@ -205,8 +216,11 @@ impl<E: Ext + 'static> ModuleImportResolver for Environment<E> {
             }
             "gr_reply" => func_instance!(Reply, ValueType::I32,
                 ValueType::I32,
-                ValueType::I64,
+                ValueType::I64,ValueType::I32,
```

Comment:
Bad code formatting

Reply:
Note that rustfmt only works with a small set of widely used macros, otherwise it leaves macro contents unchanged.